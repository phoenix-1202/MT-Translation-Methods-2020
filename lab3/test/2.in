funWithEasyCalculatingAndEasyCondition :: Int -> Integer
funWithEasyCalculatingAndEasyCondition a | a == 0 = 0
funWithEasyCalculatingAndEasyCondition a = (f (a - 1)) + 1