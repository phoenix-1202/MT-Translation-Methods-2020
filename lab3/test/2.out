#include <bits/stdc++.h>;
using namespace std;

long long funWithEasyCalculatingAndEasyCondition(int arg1) {
if ((arg1 == 0)) {
return 0;
}
return (f((arg1 - 1)) + 1);
}

//EXIT